src/lib/hmacsha1.c

```diff
@@ -10,12 +10,10 @@
 
 RCSID("$Id$")
 
+#include <openssl/hmac.h>
+#include <openssl/evp.h>
 #include <freeradius-devel/libradius.h>
```

Comment:
I notice this lacks a `#include <freeradius-devel/sha1.h>` -- would you like me to include it? I think it would be good to include.

Reply:
To answer the question, `./src/include/libradius.h` includes `sha1.h` so it'd be technically extraneous to do so. Actually, it includes `md4.h` and `sha1.h` but not `md5.h`. Go figure.